tox.ini

```diff
@@ -42,6 +42,7 @@ deps =
     -rrequirements-py3.txt
     # Extras
     Pillow
+    leveldb
```

Comment:
And same for py2, from here https://github.com/scrapy/scrapy/pull/1680/files#diff-b91f3d5bd63fcd17221b267e851608e8R15, right?